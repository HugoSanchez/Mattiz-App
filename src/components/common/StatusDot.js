import React from 'react';
import { View } from 'react-native';

// Props:
// @size: size.
// @positive: if true: color green, if not: color red.

const StatusDot = props => {  
    return (
        <View style={{ 
            width: props.size,
            height: props.size, 
            borderRadius: props.size / 2,
            backgroundColor: props.positive ? '#C4E19C' : '#C88383',
            }}
        />   
    );
}

export default StatusDot;