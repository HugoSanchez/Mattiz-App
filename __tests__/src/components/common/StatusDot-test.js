import 'react-native'
import React from 'react'
import StatusDot from '../../../../src/components/common/StatusDot'
import {render, fireEvent} from '@testing-library/react-native'

test('Renders with title', () => {
	const {container} = render(<StatusDot />)
	expect(container.children[0]).toMatchSnapshot()
})
