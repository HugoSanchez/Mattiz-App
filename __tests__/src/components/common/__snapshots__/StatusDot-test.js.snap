// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders with title 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#C88383",
      "borderRadius": NaN,
      "height": undefined,
      "width": undefined,
    }
  }
/>
`;
